src/__mocks__/datasource.ts

```diff
@@ -37,6 +37,7 @@ export const mockDatasource = new DataSource({
   type: 'athena-datasource',
   name: 'Athena Data Source',
   access: 'proxy',
+  readOnly: false,
```

Comment:
what did this do? I'm guessing this is just a new required field in 9.3.2?

Reply:
yea, new required field